can create contract.json

```diff
@@ -0,0 +1,113 @@
+    },
+    "client": {
+        "operator": {
+            "accountId": "0.0.47439",
+            "privateKey": "302e020100300506032b6570042204208f4014a3f7f7a6c7147070da98d88f9cea074c13ed0554783471825d801888cf"
+        },
+        "network": "testnet",
+        "mirrorNetwork": "testnet"
+    },
+    "test": [
+        {
+            "name": "fileCreateTransaction",
+            "request": {
+                "type": "FileCreateTransaction",
+                "data": {
+                    "keys": [ "$this.client.operatorKey" ],
+                    "contents": "$this.input.smartContractBytecode"
+                }
+            },
+            "response": {
+                "precheck": {
+                    "status": "OK"
+                }
+            }
+        },
+        {
+            "name": "fileReceipt",
+            "request": {
+                "type": "TransactionReceiptQuery",
+                "data": {
+                    "transactionId": "$this.test.fileCreateTransaction.transactionId"
+                }
+            },
+            "response": {
+                "status": "SUCCESS",
+            }
+        },
+        {
+            "name": "contractCreateTransaction",
+            "request": {
+                "type": "ContractCreateTransaction",
+                "data": {
+                    "adminKey": "$this.client.operatorKey",
+                    "gas": "2000",
+                    "bytecodeFileId": "$this.test.fileReceipt.fileId",
```

Comment:
```{
    "name": "create_file_step",
    "request": {
        ....
    },
    "response": {
        ....
    }
},
....```

Then this line should be

`"transaction_id": "step_property(create_file_step.receipt.file_id)"`